dotnet/QnAMaker/SDK-based-quickstart/Program.cs

```diff
@@ -47,15 +47,15 @@ static void Main(string[] args)
             var queryingURL = $"https://{resourceName}.azurewebsites.net";
             // </Resourcevariables>
 
-            // <TryPreview>
+            // <TryManagedPreview>
```

Comment:
As far as I know, the only doc that includes from this file is:
https://github.com/MicrosoftDocs/azure-docs-pr/blob/master/articles/cognitive-services/QnAMaker/includes/quickstart-sdk-csharp.md

Is that correct?

Currently, that doc does not refer to this snippet (either with the name "TryPreview" or "TryManagedPreview". It should probably be updated to ensure it's using all the snippets in this file.

Reply:
yes.